ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -217,6 +217,15 @@ def test_full_backup_and_restore(self):
             self.master.run_command(['ipa-restore', backup_path],
                                     stdin_text=dirman_password + '\nyes')
 
+            # check the file permssion and ownership is set to 770 and
+            # dirsrv:dirsrv after restore on /var/log/dirsrv/slapd-<instance>
+            # related ticket : https://pagure.io/freeipa/issue/7725
+            dashed_domain = self.master.domain.realm.replace(".", '-')
```

Comment:
There is a function to convert a realm into the dirsrv instance name, please use that instead:

from ipapython.ipaldap import realm_to_serverid
...
instance = realm_to_serverid(self.master.domain.realm)
log_path = paths.VAR_LOG_DIRSRV_INSTANCE_TEMPLATE % instance
...